packages/dds/tree/src/feature-libraries/treeCursorUtils.ts

```diff
@@ -0,0 +1,267 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+import {
+    FieldKey,
+    TreeType,
+    UpPath,
+    Value,
+    CursorLocationType,
+    ITreeCursorSynchronous,
+    NodeData,
+} from "../tree";
+import { fail } from "../util";
+
+/**
+ * @returns an ITreeCursorSynchronous for a single root.
+ */
+export function singleStackTreeCursor<TNode extends NodeData>(
+        root: TNode,
+        adapter: CursorAdapter<TNode>): ITreeCursorSynchronous {
+    return new StackCursor(root, adapter);
+}
+
+/**
+ * Provides functionality to allow `singleStackTreeCursor` to implement a cursor.
```

Comment:
```suggestion
 * Provides functionality to allow {@link singleStackTreeCursor} to implement a cursor.
```

Reply:
Generally, "link" uses package exported identifiers, and this type is internal. Not using link here was intentional. As the type this comment is on also is internal, I think it using link here would not break the build, but thats just because its not validated.